cvmfs/catalog_diff_tool.h

```diff
@@ -53,7 +54,8 @@ class CatalogDiffTool {
  protected:
   virtual void ReportAddition(const PathString& path,
                               const catalog::DirectoryEntry& entry,
-                              const XattrList& xattrs) = 0;
+                              const XattrList& xattrs,
+                              const FileChunkList& chunks) = 0;
```

Comment:
Don't we need that change for `ReportModification`, too?

Reply:
Good point